subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -628,34 +651,10 @@ After identifying the file(s) that are changed during each build, you should loo
 In some cases, it may be obvious (e.g., a Java file is compiled with `compileJava`).
 In other cases, you can use `--info` logging to find the task that is out-of-date due to the identified files.
 
-[[sec:continuous_build_limitations_jdk9]]
-==== Restrictions with Java 9
-
-Due to class access restrictions related to Java 9, Gradle cannot set some operating system specific options, which means that:
-
-* On macOS, Gradle will poll for file changes every 10 seconds instead of every 2 seconds.
-* On Windows, Gradle must use individual file watches (like on Linux/Mac OS), which may cause continuous build to no longer work on very large projects.
-
-[[sec:performance_and_stability]]
-==== Performance and stability
-
-The JDK file watching facility relies on inefficient file system polling on macOS (see: https://bugs.openjdk.java.net/browse/JDK-7133447[JDK-7133447]).
-This can significantly delay notification of changes on large projects with many source files.
-
-Additionally, the watching mechanism may deadlock under _heavy_ load on macOS (see: https://bugs.openjdk.java.net/browse/JDK-8079620[JDK-8079620]).
-This will manifest as Gradle appearing not to notice file changes.
-If you suspect this is occurring, exit continuous build and start again.
-
-On Linux, OpenJDK's implementation of the file watch service can sometimes miss file system events (see: https://bugs.openjdk.java.net/browse/JDK-8145981[JDK-8145981]).
-
 [[sec:changes_to_symbolic_links]]
 ==== Changes to symbolic links
 
- * Creating or removing symbolic link to files will initiate a build.
- * Modifying the target of a symbolic link will not cause a rebuild.
- * Creating or removing symbolic links to directories will not cause rebuilds.
- * Creating new files in the target directory of a symbolic link will not cause a rebuild.
- * Deleting the target directory will not cause a rebuild.
+In general, Gradle will not detect changes to symbolic links or to files in the target of symbolic links.
```

Comment:
"or to files in the target of symbolic links." -> "or to files reference via symbolic links".